components/component-stylesheets/avatar.css

```diff
@@ -0,0 +1,46 @@
+.avatar{
+    border-radius: 50%;
+    width: 8rem;
+    height: 8rem;
```

Comment:
offload the width and height to another class.

use combination of classes `.avatar .avatar-s` for - small avatar.

Reply:
changes made in the recent commit.